src/ontology/cl-edit.owl

```diff
@@ -28558,13 +28558,13 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:10845061"^^xsd:string) r
 AnnotationAssertion(rdfs:label obo:CL_4023187 "koniocellular cell")
 EquivalentClasses(obo:CL_4023187 ObjectIntersectionOf(obo:CL_0000540 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0013615)))
 
-# Class: obo:CL_4023211 (perineurial cells)
+# Class: obo:CL_4023211 (perineurial cell)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "NCIT:C41442"^^xsd:string) obo:IAO_0000115 obo:CL_4023211 "A myofibroblast cell that belongs to the supporting tissue surrounding a nerve fiber bundle. It has thin long bipolar cytoplasmic processes, pinocytotic vesicles, fragments of external lamina and/or external lamina-like material, attachment plaques, and desmosome-like junctions.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "NCIT:C41442"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:25818566"^^xsd:string) obo:IAO_0000115 obo:CL_4023211 "A myofibroblast cell that belongs to the supporting tissue surrounding a nerve fiber bundle. It has thin long bipolar cytoplasmic processes, pinocytotic vesicles, fragments of external lamina and/or external lamina-like material, attachment plaques, and desmosome-like junctions.")
 AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> obo:CL_4023211 "https://orcid.org/0000-0001-7655-4833")
```

Comment:
Using dce:contributor and the orcid is a string :)

Reply:
whoops soz, need to fix my protege settings for that >.<

Reply:
only if it's not an orcid for dc:terms

Reply:
ah we didn't implement the "allowed" annotation properties in CL then

Reply:
We did. I'll investigate it.